packages/schematics/src/command-line/shared.ts

```diff
@@ -51,6 +56,22 @@ function parseDashDashFiles(dashDashFiles: string): string[] {
   return f.split(',').map(f => f.trim());
 }
 
+function getFilesFromLocalChanges(staged: boolean): string[] {
+  if (staged) {
+      return execSync(`git diff --name-only --cached`)
```

Comment:
Some code duplication. Maybe create a `getFilesNamesFromGitOutput` helper?

Reply:
Better yet, `simple-git` handles this more nicely. It gives nice JSON format results from git commands. https://github.com/steveukx/git-js. If we want to bring in that dependency